mirrord/layer/tests/apps/recv_from/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use std::{mem::MaybeUninit, net::ToSocketAddrs};
+
+use socket2::{Domain, SockAddr, Socket, Type};
+
+fn main() {
+    let google_dns_server = "8.8.8.8:53";
```

Comment:
I think you can just use `parse`

Reply:
there is no FromStr impl for SockAddr, cant use parse ):

Reply:
```
let address: SocketAddr = "1.2.3.4".parse().unwrap();
socket.connect(&address.into())?;
```

Reply:
thanks!